src/shared/actions/events/event.js

```diff
@@ -32,6 +32,12 @@ export function fetchFailure(error) {
 
 export const fetchEvent = (fetch) => (
   (dispatch, getState, nextState) => {
+    const storedEvent = getState().event;
+
+    if (storedEvent && Object.keys(storedEvent).length > 0) {
```

Comment:
I feel that this case is already handled here:
https://github.com/redbadger/website-next/blob/master/src/shared/actions/events/index.js#L35

?